charts/openfaas-cloud/templates/ingress/ingress-auth.yml

```diff
@@ -0,0 +1,26 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: openfaas-auth-ingress
+  namespace: {{ .Values.global.coreNamespace }}
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
```

Comment:
Just a thought, would it make sense to template the ingress class? If not now, in a future PR/issue?